src/mmw/js/src/core/views.js

```diff
@@ -203,6 +210,45 @@ var MapView = Marionette.ItemView.extend({
         }
     },
 
+    aoiChangeWarning: _.debounce(function() {
```

Comment:
Cool use of `_.debounce` here, I didn't know we could define view methods this way.